redaxo/src/addons/structure/plugins/history/package.yml

```
@@ -20,4 +20,4 @@ requires:
     packages:
         structure: '^2.2.0'
         structure/content: '^2.2.0'
-    redaxo: '^5.4'
+    redaxo: '^5.14.2'
```

So ließe es sich aktuell im Repo-Stand dann wahrscheinlich nicht mehr installieren, weil der Core noch auf 5.14.1 steht.
Es muss also die Core-Version dann auch tatsächlich angehoben werden, damit es installierbar bleibt.
Beide Versionen sollten `-dev` enthalten.

Insgesamt ist für mich die Frage, ob wir es nicht besser in 5.15 nehmen sollten. Weil es fixt ja nichts, und ich versuche in Bugfix-Releases Code-Änderungen, die keinen Bug fixen, eher zu vermeiden. (Docs hingegen sind natürlich kein Problem.)

Kurzum: Ich würde hier im PR die Core-Version auf `5.15.0-dev` setzen, und hier dann:

```suggestion
    redaxo: '^5.15.0-dev'
```

Im Mutter-Addon würde ich die Mindestversion dann aber auch anheben. Sonst aktualisiert vielleicht jemand in niedrigerer Core-Version das structure-Addon, und kann dann aber das history-plugin nicht mehr installieren, wenn er das dann irgendwann nutzen möchte.

---

Bugfixes würden wir ab dann wieder in den bugfix-branch nehmen, damit ich noch die Option erhalte, ggf. 5.14.2 irgendwann rauszubringen.